Aptoide/aptoide-client-v8
# [ASV-572] horizontal scroll trending bundles

**What does this PR do?**

  Adds horizontal scroll to bundles that have other bundles.
  Removes bundles from the list that don't have any app in it.

**Questions**

AdultContentFilter is not being used in StoreTabFragmentChooser at the moment. I researched where it was being passed as true and there were 2 places.
One of them I'm almost sure it's not being used anymore, in StorePagerAdapter (search for API).
The other one is in DeeplinkManager in GenericDeeplink.
This boolean is used to create a displayable of a switch 
Do you know if these are really being used specially the second one?

**Database changed?**

   No

**Where should the reviewer start?**

RemoteBundleDataSource.java
BundleRepository.java
MoreBundleFragment.java
MoreBundlePresenter.java

**How should this be manually tested?**

Navigate trough the Trending and Top Games and other bundles that have more bundles inside them
Check if Home is also working correctly

**What are the relevant tickets?**

https://aptoide.atlassian.net/browse/ASV-572 (task)
https://aptoide.atlassian.net/browse/ASV-311 (user-story)

**Code Review Checklist**
